Source/DeckPresentationController.swift

```diff
@@ -696,5 +723,28 @@ final class DeckPresentationController: UIPresentationController, UIGestureRecog
         
         return true
     }
-    
+
+	func expandView() {
+
+		presentedViewController.view.constraints
+			.filter({ $0.firstItem === presentedViewController.view && $0.firstAttribute == .height })
+			.forEach({ $0.constant = containerView?.frame.height ?? UIScreen.main.bounds.height })
+
+		UIView.animate(withDuration: Constants.defaultAnimationDuration,
+					   delay: 0,
+					   options: .curveEaseInOut,
+					   animations: containerView?.layoutIfNeeded ?? presentedViewController.view.layoutIfNeeded)
+	}
+
+	func compressView() {
+
+		presentedViewController.view.constraints
+			.filter({ $0.firstItem === presentedViewController.view && $0.firstAttribute == .height })
+			.forEach({ $0.constant = modalHeight })
+
+		UIView.animate(withDuration: Constants.defaultAnimationDuration,
```

Comment:
Is it worth moving this to an `animateView()` function rather than repeating the same code twice?

Reply:
Nice catch, done on 594da87